### ISSUE TITLE
Theme is not animated

### ISSUE THREAD

**Issue by Yoruiopz** [JulienMaille/spicetify-dynamic-theme#75]
**Describe the bug**
Background is not animated as shown in the preview.

**To Reproduce**
1. Use the theme and play a song.

**Screenshots**

**Desktop Setup**
- Operating System: Windows 11
- Spotify: 1.2.37.701
- Spicetify: 2.36.11
- Theme: Default Dynamic

**Comment by JulienMaille** [comment]
Please see #74 it might be the similar issue.

**Comment by Yoruiopz** [comment]
> Please see #74 it might be the similar issue.

This is a different issue. The BG is animated in your preview, in my client its a static image with blur on it, additionally the change between the songs doesnt fade like how it should in your preview, instead it instantly changes it.

**Comment by JulienMaille** [comment]
please paste the errors you see in the Console (as explained in the bug report template)

**Comment by Yoruiopz** [comment]
> please paste the errors you see in the Console (as explained in the bug report template)

`index.html:1 document.domain mutation is ignored because the surrounding agent cluster is origin-keyed.
4151.css:1 
 Failed to load resource: net::ERR_NAME_NOT_RESOLVED
favicon.ico:1 
 Failed to load resource: net::ERR_NAME_NOT_RESOLVED
extension.js:8 Initializing Spicetify Marketplace v1.0.1
extension.js:8 Loaded Marketplace extension
vendor~xpui.js:1 Uncaught (in promise) Error: enqueueSnackbar called with invalid argument
    at Object.enqueueSnackbar (vendor~xpui.js:1:1704751)
    at Spicetify.showNotification (spicetifyWrapper.js:741:24)
    at e (extension.js:10:774)
xpui.js:1 cosmos-connector:  fetchAdForSlot endpoint=%o sp://ads/v1/ads/hpto
cdn.jsdelivr.net/gh/…ime=1716043886320:1 
 Failed to load resource: the server responded with a status of 403 ()
index.html:1 Refused to execute script from 'https://cdn.jsdelivr.net/gh/surfbryce/beautiful-lyrics@main/dist/beautiful-lyrics.js?time=1716043886320' because its MIME type ('text/plain') is not executable, and strict MIME type checking is enabled.
gew1-spclient.spotif…/v3/configuration:1 
 Failed to load resource: the server responded with a status of 404 ()
vendor~xpui.js:1 provider:transport resolve error 
k
 
Object
index.html:1 Uncaught (in promise) 
k
﻿

`

**Comment by JulienMaille** [comment]
Have you tried reinstalled like other did in  #74 ?
Which theme have you selected in the Marketplace menu?
![image](https://github.com/JulienMaille/spicetify-dynamic-theme/assets/182520/8ce9e9c3-f287-4fbb-9281-22e03d7fd442)

**Comment by Yoruiopz** [comment]
> Have you tried reinstalled like other did in #74 ? Which theme have you selected in the Marketplace menu? ![image](https://private-user-images.githubusercontent.com/182520/331812686-8ce9e9c3-f287-4fbb-9281-22e03d7fd442.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYwNDgzMTYsIm5iZiI6MTcxNjA0ODAxNiwicGF0aCI6Ii8xODI1MjAvMzMxODEyNjg2LThjZTllOWMzLWYyODctNGZiYi05MjgxLTIyZTAzZDdmZDQ0Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxOFQxNjAwMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yY2NjM2NmNDM0YTNlNThhOWI2YzRiOGRjOGRhODQ3NmJmMmY0ZmYwMzk2MjFkM2ZlN2NhZDdjZDhjODg5OWQwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.dwgQVTLWFLPjDviHlMSxuwF0ltv0JNOYvKseNJMFzeU)

Yes, i tried following the steps to resolve the issue the person in that thread was facing, it did nothing.

I have the "Base" selected on that dropdown menu

**Comment by JulienMaille** [comment]
Do you have anything else installed from the Marketplace?

**Comment by ohitstom** [comment]
> Do you have anything else installed from the Marketplace?

hi, same issue here

**Comment by JulienMaille** [comment]
This class still exists on artists pages.
I have the exact same version as you do and can't reproduce the issue ...
![image](https://github.com/JulienMaille/spicetify-dynamic-theme/assets/182520/8d159313-7c98-4ae8-adba-bda583669854)

**Comment by Yoruiopz** [comment]
> Do you have anything else installed from the Marketplace?

Yes, beautiful Lyrics only

**Comment by ohitstom** [comment]
this issue is no longer present for me, it seemed like some sort of playerapi issue 👍🏻

**Comment by Yoruiopz** [comment]
> this issue is no longer present for me, it seemed like some sort of playerapi issue 👍🏻

how did you fix it?

**Comment by ohitstom** [comment]
> > this issue is no longer present for me, it seemed like some sort of playerapi issue 👍🏻
> 
> how did you fix it?

i reinstalled spotify which made my player state empty then when i played a song everything started working - thats all i know though

**Comment by Yoruiopz** [comment]
@JulienMaille Let us know if you find a solution, it is still persisting for me even after 3+ updates to spicetify

**Comment by JulienMaille** [comment]
> vendor~xpui.js:1 Uncaught (in promise) Error: enqueueSnackbar called with invalid argument
>     at Object.enqueueSnackbar (vendor~xpui.js:1:1704751)
>     at Spicetify.showNotification (spicetifyWrapper.js:741:24)
>     at e (extension.js:10:774)

I think you have something wrong with your setup @Yoruiopz

**Comment by Yoruiopz** [comment]
> > vendor~xpui.js:1 Uncaught (in promise) Error: enqueueSnackbar called with invalid argument
> > at Object.enqueueSnackbar (vendor~xpui.js:1:1704751)
> > at Spicetify.showNotification (spicetifyWrapper.js:741:24)
> > at e (extension.js:10:774)
> 
> I think you have something wrong with your setup @Yoruiopz

The `enqueueSnackbar` error, indicates that an invalid argument is being passed to the function, suggesting an issue within the code implementation rather than my setup.

It seems the arguments passed to `enqueueSnackbar` are either missing, malformed, or of incorrect types. I've tried reinstalling Spotify and Spicetify and ensuring the "Base" theme is selected from the Marketplace menu, but the issue persists.

**Comment by JulienMaille** [comment]
Where is the call to `enqueueSnackbar` located? It's not in this theme.
Have you tried removing beautiful Lyrics and Eternal Jukebox to see if it helps?

**Comment by Yoruiopz** [comment]
> Where is the call to `enqueueSnackbar` located? It's not in this theme. Have you tried removing beautiful Lyrics and Eternal Jukebox to see if it helps?
> 
> EDIT: I believe this comes from the Marketplace ![image](https://private-user-images.githubusercontent.com/182520/341507423-065f3d69-5fbc-4145-997f-e55c4b085f57.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5MzE2MDIsIm5iZiI6MTcxODkzMTMwMiwicGF0aCI6Ii8xODI1MjAvMzQxNTA3NDIzLTA2NWYzZDY5LTVmYmMtNDE0NS05OTdmLWU1NWM0YjA4NWY1Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMVQwMDU1MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZWZiNzFhYjllOWM1MmI4OTc1NTQ4MTg0Y2FlYzZlYTU4MDRlNTlkNWIyZDc3NTI5MzRiYTdhNzUwYTQxNzEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.mJWrb8-8ZCXolpWnPZ2Frf_m1Q4CXSkvWSJq8erz_H8)

I tried removing Beautiful Lyrics and Eternal Jukebox, but the issue persists.

**Comment by JulienMaille** [comment]
Have you tried to update/reinstall the marketplace?

**Comment by Yoruiopz** [comment]
> Have you tried to update/reinstall the marketplace?

Yes, the issue still persists.

**Comment by Yoruiopz** [comment]
Additionally, when switching songs, the background doesn't transition smoothly. It abruptly cuts to the blurred and zoomed-in version of the new cover art.

**Comment by Yoruiopz** [comment]
I read through your code and noticed a few potential areas that might be causing the issues. The abrupt transitions between songs and light/dark modes, as well as the static background image, might be due to the lack of CSS transitions for elements that change colors and properties. This isn't an end user setup issue.

I'm not sure if the preview posted on your repo is of a prerelease build of the theme, but these features are not implemented in the final build that users receive through the marketplace.

**Comment by Yoruiopz** [comment]
After further troubleshooting, I found a solution in the Spicetify Discord. The issue was caused by having "Animation Effects" turned off in my Windows settings under Accessibility. While I believe having this feature on negatively impacts performance and makes Windows appear sluggish, it seems that Windows overrides the animation properties listed in your CSS when this setting is disabled, which I find quite stupid to be honest considering Spotify is a standalone application.

I discovered this by noticing the lack of animation in other snippets, prompting me to investigate further.

I hope this helps anyone encountering the same issue in the future.
